include/private/rdycoreimpl.h

```diff
@@ -176,6 +177,10 @@ struct _p_RDy {
     CeedScalar           dt;
 
   } ceed_rhs;
+
+  RiemannDataSWE datal_internal_edges, datar_internal_edges;
```

Comment:
If you think it's helpful and not too much typing, consider placing these into a struct variable like `ceed_rhs` above (e.g. named `data_swe` or `swe_fluxes`).